lidar_freq: 10.0
azim_fov: 360.0
azim_res: 5.0
elev_fov: 2.0
elev_res: 5.0
range_min: 0.025
range_max: 10.0
set_max_inf: true
use_rollpitch: true